src/Akka.CQRS.Tests.Hosting/AkkaConfiguration.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Akka.Actor;
+using Akka.Cluster.Hosting;
+using Akka.CQRS.Infrastructure;
+using Akka.CQRS.Pricing.Actors;
+using Akka.CQRS.TradeProcessor.Actors;
+using Akka.Hosting;
+using Akka.Persistence.Query.Sql;
+using Akka.Persistence.SqlServer.Hosting;
+using Akka.Remote.Hosting;
+using Akka.Persistence.Query;
+using Akka.Util;
+using Akka.Cluster.Sharding;
+namespace Akka.CQRS.Tests.Hosting;
+
+public static class AkkaConfiguration
+{
+    public static AkkaConfigurationBuilder ConfigureTradeProcessor(this AkkaConfigurationBuilder builder, string connectionString)

```

Comment:
* Production configuration and test configuration should be composed from the same underlying methods - that way the test covers production config. This is a totally separate configuration here.
* Each service's configuration should be composed from individual methods - I should be able to start the actors for the TradeProcessor without also having to configure how Akka.Remote works, for instance.